lib/clone_kit/cloners/active_record_merging_ruleset_cloner.rb

```diff
@@ -5,6 +5,8 @@
 module CloneKit
   module Cloners
     class ActiveRecordMergingRulesetCloner < ActiveRecordRulesetCloner
+      ID = "id".freeze
```

Comment:
If we add `# frozen_string_literal: true` to the top of the file, then we don't need to call `freeze` directly here.

Reply:
Also, it looks like this was previously a symbol where it was used.  Is there any danger in changing?  Or is this a necessary change to support both Mongoid and ActiveRecord?